obj.js

```diff
@@ -355,13 +359,13 @@ async function sendClassDescription(className, instance) {
     if (propName.startsWith("_") || propName == "constructor") {
       continue;
     }
-    if (typeof(proto[propName]) == "function") {
+    let property = Object.getOwnPropertyDescriptor(proto, propName);
+    if (typeof(property.value) == "function") {
```

Comment:
dito

Reply:
This was a bug fix from PR#1 - it would run the getter, when what you really wanted to do is to see whether this was a method.